nvflare/edge/tools/edge_recipe.py

```diff
@@ -28,6 +28,26 @@
 
 
 class ModelManagerConfig:
+    """Configuration class for the model manager in federated learning.
+
+    This class configures how the model manager handles model updates, versioning,
+    and aggregation strategies for federated learning workflow.
+
+    Attributes:
+        max_num_active_model_versions: Maximum number of active model versions
+            that can be processed for the current model version. Default: 3
+        max_model_version: Maximum model version number before stopping training.
```

Comment:
maybe need to add explanation somewhere saying this can not be 1 or add validation that this needs to be >=2